src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -19,6 +19,17 @@
 
 REPO_CENTRAL = b'https://hg.mozilla.org/mozilla-central'
 REPO_REVIEW = b'https://reviewboard-hg.mozilla.org/gecko'
+MAX_COMMENTS = 30
+MOZREVIEW_COMMENT_SUCCESS = '''
+Static analysis didn't find any C/C++ defects in this patch. Hooray!
```

Comment:
Nit: (same as below) please move `C/C++` to the beginning of the sentence, e.g.:

```
C/C++ static analysis didn't find any defects in this patch. Hooray!
```